presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -35,6 +35,13 @@ Array Functions
     array is empty); ``NULL`` if the predicate function returns ``NULL`` for one or more elements and ``false``
     for all other elements.
 
+.. function:: array_average(array(T)) -> double
```

Comment:
You might want to just say `array_average(array<double>) -> double` instead, rather than saying it's a generic type T that's coercible to double. This would be more consistent with code. Also, for `map_normalize` you just used `map<varchar,double>` rather than saying `map<varchar,T>`. People should know that you can all a function that takes `array<double>` with `array<T coercible with double>`.